h2o-r/.gitignore

```
@@ -5,7 +5,8 @@ h2o-package/inst/branch.txt
 h2o-package/inst/source_code_repository_info.txt
 h2o-package/NAMESPACE
 
-R/
+R/README.md
```

I also have the h2o_package.pdf in that folder, maybe that should be ignored to?
```suggestion
R/README.md
R/h2o_package.pdf
```

---

Thank you - there might be more - I probably don't run the full build locally but jenkins should tell me - I will use your suggestion, thanks